cime_config/config_grids.xml

```diff
@@ -3935,12 +3934,22 @@
       <map name="ATM2WAV_SMAPNAME">cpl/gridmaps/T62/map_T62_TO_wQU100_bilin.20210202.nc</map>
     </gridmap>
 
-    <gridmap atm_grid="T62" wav_grid="wDE4">
-      <map name="ATM2WAV_SMAPNAME">cpl/gridmaps/T62/map_T62_TO_wDE4_blin.20200611.nc</map>
+    <gridmap atm_grid="T62" wav_grid="wQU225EC60to30">
```

Comment:
are you planning to support jigsaw grids eventually?  I only see a v1 grid compset

Reply:
Yes definitely. I just did Ec60to30 to start.